.golangci.yml

```diff
@@ -36,37 +36,38 @@ run:
   tests: true
 
 linters:
-  disable-all: true
-  enable:
-    - asciicheck
-    - bodyclose
-    - deadcode
-    - dogsled
-    - dupl
-    - errcheck
-    - gci
-    - goconst
-    - gocritic
-    - gocyclo
-    - godot
-    - gofmt
-    - gofumpt
-    - goimports
-    - goprintffuncname
-    - gosec
-    - gosimple
-    - govet
-    - ineffassign
-    - misspell
-    - prealloc
-    - revive
-    - staticcheck
-    - structcheck
-    - unconvert
-    - unparam
-    - unused
-    - varcheck
+  enable-all: true
+  disable:
+   - wrapcheck
+   - varnamelen
+   - thelper
+   - tagliatelle
+   - paralleltest
+   - nlreturn
+   - exhaustivestruct
+   - wsl
+   - goerr113
```

Comment:
handling this would be nice overall

Reply:
yes, keep for the next iteration https://github.com/GetStream/stream-chat-go/issues/9